autogluon/utils/tabular/features/generators/pipeline.py

```diff
@@ -108,9 +108,9 @@ def print_feature_metadata_info(self, log_level=20):
             self._log(log_level, f'\t\tThese features do not need to be present at inference time for this FeatureGenerator.')
         if self._feature_metadata_in_unused.get_features():
             self._log(log_level, f'\tUnused Original Features (Count: {len(self._feature_metadata_in_unused.get_features())}): {self._feature_metadata_in_unused.get_features()}')
-            self._log(log_level, f'\t\tThese features were not valid input to any of the feature generators. Add a feature generator compatible with these features to utilize them.')
+            self._log(log_level, f'\t\tThese features were not used to generate any of the output features. Add a feature generator compatible with these features to utilize them.')
```

Comment:
IMO the print statements should be more from the perspective of the overall ML pipeline, rather than from the perspective of a FeatureGenerator (because many users who just skim docs may have no idea what this is or what "output features" are).  So this could say for example:
```
These features will be ignored by all models. Add a feature generator compatible with these features to utilize them.
```

Reply:
Also if we cannot easily track the reason why each feature was ignored, perhaps should clarify here why features may be ignored in general (eg. due to taking constant value across all rows).

Reply:
I'd prefer to keep it in context of generator to avoid assumptions about what the data is used for. Happy to tweak this later though. Added some extra logs on common reasons for being unused and TODO to try to add logic to determine why a feature was unused.